#goal
int32 pan_start
int32 pan_step
int32 pan_end
int32 tilt_start
int32 tilt_step
int32 tilt_end
---
#result
bool success
---
#feedback
sensor_msgs/JointState feedback_ptu_pose
